app-backend/common/src/main/scala/Authentication.scala

```diff
@@ -127,7 +142,7 @@ trait Authentication extends Directives {
         onSuccess(query.transact(xa).unsafeToFuture).flatMap {
           case MembershipAndUser(plat, org, userUpdate) =>
             (plat, org) match {
-              case (Platform(_, _, _, true, _), Organization(_, _, _, _, _, true, _, _, _)) =>
+              case (Some(Platform(_, _, _, true, _)), Some(Organization(_, _, _, _, _, true, _, _, _))) =>
```

Comment:
It may be worth matching on the specific fields you care about, since you only care about one from the org and one from the platform, to decrease the cognitive burden of tracking how many `_`s you've gone by and to make meaning more obvious, e.g.

`(plat map { _.isActive }, org map { _.isActive }) match { ... }`

Reply:
:+1: